lib/tests/streamlit/web/server/server_test.py

```diff
@@ -13,66 +13,60 @@
 # limitations under the License.
 
 """Server.py unit tests"""
+
 import asyncio
 import errno
 import os
 import shutil
 import tempfile
 import unittest
 from unittest import mock
-from unittest.mock import MagicMock, patch
+from unittest.mock import patch
 
 import pytest
 import tornado.httpserver
+import tornado.httpserver
+import tornado.httpserver
 import tornado.testing
+import tornado.testing
+import tornado.testing
+import tornado.web
 import tornado.web
+import tornado.web
+import tornado.websocket
+import tornado.websocket
 import tornado.websocket
 
 import streamlit.web.server.server
-from streamlit import config, RootContainer
-from streamlit.cursor import make_delta_path
-from streamlit.elements import legacy_data_frame as data_frame
-from streamlit.runtime.forward_msg_cache import ForwardMsgCache
-from streamlit.runtime.forward_msg_cache import populate_hash_if_needed
+from streamlit import config
 from streamlit.logger import get_logger
 from streamlit.proto.ForwardMsg_pb2 import ForwardMsg
+from streamlit.runtime.forward_msg_cache import populate_hash_if_needed
 from streamlit.runtime.uploaded_file_manager import UploadedFileRec
 from streamlit.watcher import event_based_path_watcher
-from streamlit.web.server.server import DebugHandler
-from streamlit.web.server.server import HealthHandler
 from streamlit.web.server.server import MAX_PORT_SEARCH_RETRIES
-from streamlit.web.server.server import MessageCacheHandler
 from streamlit.web.server.server import RetriesExceeded
 from streamlit.web.server.server import Server
 from streamlit.web.server.server import State
-from streamlit.web.server.server import StaticFileHandler
 from streamlit.web.server.server import start_listening
```

Comment:
nit: not quite part of this change, but might as well merge imports as possible here